src/test/resources/pages/instructorFeedbackAddSuccess.html

```diff
@@ -986,7 +992,8 @@ <h5 class="col-sm-2">
                   <input id="noofchoicecreated--1" name="noofchoicecreated" type="hidden" value="2">
                 </div>
                 <div class="col-sm-6 col-lg-5 col-lg-offset-1 padding-right-25px">
-                  <div class="border-gray narrow-slight visible-xs margin-bottom-7px margin-top-7px"></div>
```

Comment:
Indeed, it seems like someone did not use God mode when modifying these files before. Let changes like this pass.